proposals/0000-borrow-operator.md

```diff
@@ -0,0 +1,195 @@
+# Borrow Operator, Pass-by-Immutable-Borrow, and Pass-by-Value
```

Comment:
"Pass-by-Value" should not be in the title, since that's not something you're proposing.

Reply:
The reason why I included it in the title is that I wanted to contrast it with Pass-by-Immutable borrow. That being said, I agree that it can be removed.